src/OpenSage.Game/Gui/DebugUI/DebugOverlay.cs

```diff
@@ -32,32 +32,32 @@ public DebugOverlay(Scene3D scene3D, ContentManager contentManager)
             _debugDrawables = new List<IDebugDrawable>();
         }
 
-        public void AddPoint(Vector3 point, ColorRgbaF color, float? duration = null)
+        public void AddPoint(in Vector3 point, ColorRgbaF color, float? duration = null)
```

Comment:
Looks great - just these `ColorRgbaF` parameters should also have the `in` modifier, then it's ready to merge.